server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -193,6 +193,59 @@ function checkThrottleAndUsage(
     }
 }
 
+/**
+ * This function will store total connection count per node and per cluster in Redis, and
+ * will also log these counters using ConnectionCountPerNode and TotalConnectionCount metrics respectively.
+ */
+async function logConnectionCount(isConnect: boolean, cache?: core.ICache): Promise<void> {
+    const connectionCountPerNodeMetric = Lumberjack.newLumberMetric(LumberEventName.ConnectionCountPerNode);
+    const totalConnectionCountMetric = Lumberjack.newLumberMetric(LumberEventName.TotalConnectionCount);
+    const nodeName = process.env.NODE_NAME ?? "nodeName";
```

Comment:
Why not crate a class to hold the `cache` and `nodeName` there ?

Reply:
Hey Xin. I'm not sure I understood your suggestion and what benefit we will get with that over this.

Reply:
So you are passing a `cache` reference  everytime for this call invocation, and you have to evaluate `nodeName` every time when calling this method.

Both these two information, are some internal property of the the `ConnectionCountLogger`, where it should be self isolated but not passed in / calculate every time, and they stays unchanged, for example, you won't expect another `ICache` later when program is running, and `nodeName` won't change.

In fact, the cache is only used by this `ConnectionCountLogger` at this moment, so for single responsibility and encapsulation, I think it worse to create a class to wrap the logic